common/appe-appliance-console-cli.adoc

```diff
@@ -0,0 +1,410 @@
+:numbered!:
+
+Currently, the `appliance_console_cli` feature is a subset of the full functionality of the `appliance_console` itself, and covers functions most likely to be scripted using the command-line interface (CLI). 
+
+. After starting the {product-title} appliance, log in with a user name of `root` and the default password of `smartvm`. This displays the Bash prompt for the root user. 
+. Enter the `appliance_console_cli` or `appliance_console_cli --help` command to see a list of options available with the command, or simply enter `appliance_console_cli --option <argument>` directly to use a specific option.
+
+.Database Configuration Options
+[cols="1,1", frame="all"]
+|===
+|
+
+							Option
+
+
+|
+
+							Description
+
+|
+
+							--region (-r)
+
+
+|
+
+							create a new region in the database (requires database credentials passed)
```

Comment:
This feels like it should more clearly state that the region number is given as an argument for this option.